loadbalancer/algorithm_test.go

```diff
@@ -43,7 +43,7 @@ func TestSelectAlgorithm(t *testing.T) {
 
 		if len(rr[0].LBEndpoints) != 1 ||
 			rr[0].LBEndpoints[0].Scheme != "https" ||
-			rr[0].LBEndpoints[0].Host != "www.example.org" ||
+			rr[0].LBEndpoints[0].Host != "www.example.org:443" ||
```

Comment:
I guess this is fine, because it does not change the outgoing request nor change the metrics